pkg/sql/codegen/pai/codegen.go

```diff
@@ -60,10 +60,9 @@ func FormatCkptDir(modelName string) (string, error) {
 	if len(ossURIParts) != 2 {
 		return "", fmt.Errorf("SQLFLOW_OSS_CHECKPOINT_DIR must be of format: oss://bucket/?role_arn=xxx&host=xxx")
 	}
-	ossDir := strings.Join([]string{strings.TrimRight(ossURIParts[0], "/"), modelName}, "/")
+	ossDir := filepath.Join(ossURIParts[0], modelName)
```

Comment:
The package `filepath` is used to deal with different operating system paths, for OSS path, it's always `/`.